### DISCUSSION TITLE
Latest update

### DISCUSSION THREAD

**Discussion by Lonewolf71** [breezy-weather/breezy-weather#789] [Q&A] [answered]
Hello, I'm having issues with the latest version.
When I attempt to refresh by swiping down, nothing happens. It also says Accuweather isn't available anymore!?

It also says "please wait a few seconds before retrying"

**Comment by papjul** [comment, marked as answer]
It won't be able to refresh if you installed a flavor of Breezy Weather without support for AccuWeather.

You need to go to bottom of your location > Edit > Weather sources and decide what you want to do from there (install a different flavor or change the sources for your location).

**Reply by Lonewolf71** [reply]
It was working great until the new update.
Why does it only show Automatic and Open Meteo now?

**Reply by papjul** [reply]
I don't know, you're the one who switched to the flavor of Breezy Weather without AccuWeather support. You can go back to the standard flavor at any time.

**Reply by papjul** [reply]
To make it more clear, I will add the following tips in next version of Breezy Weather when "Source no longer available" error happens:
1. This can happen when you added your location in the Standard flavor of Breezy Weather and then switched to the F-Droid one which has less weather sources supported. Going back to the Standard flavor may solve this problem.
2. Either change the weather sources for this location or delete and re-add this location.

**Reply by Lonewolf71** [reply]
Thank you
